### PR TITLE
Add common props and target

### PR DESCRIPTION
**Pull request by andreasohlund**
Just some minor cleanup based on what we learned in https://github.com/Particular/DotNetCoreLaunch/issues/7

### CODE REVIEW

**Comment by andreasohlund** [comment]
@timbussmann starting to wonder if we actually need all this given that msmq will never build for netcore? (perhaps we only need a subset)

**Comment by timbussmann** [comment]
well, I think it's easier to just stay to the common approach than some customized config? Nothing on these changes seems to be tied to .net core?

**Comment by andreasohlund** [comment]
Was thinking that most of the workarounds where releated to GitVersion + tests building/running on netcore but I agree that a common approach has merits.

@Particular/msmq-transport-maintainers please review

File: src/Directory.Build.targets [hunk 1]
```diff
@@ -0,0 +1,22 @@
+<Project>
+
+  <PropertyGroup>
+    <Authors>NServiceBus Ltd</Authors>
+    <Company>NServiceBus Ltd</Company>
+    <PackageLicenseUrl>http://particular.net/LicenseAgreement</PackageLicenseUrl>
+    <PackageRequireLicenseAcceptance>true</PackageRequireLicenseAcceptance>
+    <Copyright>Copyright 2010-$([System.DateTime]::UtcNow.ToString(yyyy)) NServiceBus. All rights reserved</Copyright>
+    <PackageTags>nservicebus servicebus msmq cqrs publish subscribe</PackageTags>
```
Comment (ramonsmits) [inline src/Directory.Build.targets]: Do we need the tags 'servicebus msmq cqrs publish subscribe' in there? Shouldn't that be 'nservicebus transport msmq'?

File: src/NServiceBus.Transport.Msmq.Tests/MsmqQueueCreatorTests.cs [hunk 2]
```diff
@@ -165,12 +165,9 @@ public void Should_not_add_everyone_and_anonymous_to_already_existing_queues()
 
             var existingQueue = GetQueue(testQueueNameForReceiving);
 
-            MessageQueueAccessRights? nullBecauseRevoked;
-            AccessControlEntryType? accessControlEntryType;
 
-            Assert.False(existingQueue.TryGetPermissions(LocalEveryoneGroupName, out nullBecauseRevoked, out accessControlEntryType));
-            Assert.False(existingQueue.TryGetPermissions(LocalAnonymousLogonName, out nullBecauseRevoked, out accessControlEntryType));
-            Assert.IsNull(accessControlEntryType);
+            Assert.False(existingQueue.TryGetPermissions(LocalEveryoneGroupName, out MessageQueueAccessRights? _, out AccessControlEntryType? _));
```
Comment (ramonsmits) [inline src/NServiceBus.Transport.Msmq.Tests/MsmqQueueCreatorTests.cs]: Nice! Didn't know this is now possible :-)

File: src/Directory.Build.targets (hunk 1 above)
Comment (andreasohlund) [inline src/Directory.Build.targets]: Updated

Comment (indualagarsamy) [inline src/Directory.Build.targets]: I don't see anything wrong with the tags. I think these are just generic tags.

**Pull request merged**